pkg

```diff
@@ -247,7 +262,7 @@ while (( "$#" )); do
 		# where <service> is the name of the specific service to be chained. The file may be a symlink to another
 		# compose file, in which case the symlink will be followed before it is processed.
 		if [ -e "$COMPOSE_FILE.$1" ] ; then
-			$SELF -f $(realpath -e "$COMPOSE_FILE.$1") $([ "$verbose" == 0 ] || echo "-v") $([ "$quiet" == 0 ] || echo "-q") $([ "$debug" == 0 ] || echo "-d") $([ "$build" == 0 ] || echo "-b") $("${COMPOSECMD[@]}" -f $(realpath -e "$COMPOSE_FILE.$1") config --services)
+			$SELF -f $(realpath "$COMPOSE_FILE.$1") $([ "$verbose" == 0 ] || echo "-v") $([ "$quiet" == 0 ] || echo "-q") $([ "$debug" == 0 ] || echo "-d") $([ "$build" == 0 ] || echo "-b") $("${COMPOSECMD[@]}" -f $(realpath "$COMPOSE_FILE.$1") config --services)
```

Comment:
I see the different with the result of realpath $COMPOSE_FILE.$1 (./infrastructure/docker/build/docker-compose-opt.yml.ats) when you add realpath function is:
/Users/ntheanh201/Workspace/apache/trafficcontrol/infrastructure/docker/build/docker-compose-opt.yml.ats

vs. the result of realpath in master branch:
/Users/ntheanh201/Workspace/apache/trafficcontrol/traffic_server/_tsb/docker-compose.yml

Reply:
This should be fixed in 7d9273d22a.

Reply:
<img width="319" alt="image" src="https://github.com/apache/trafficcontrol/assets/29043139/2d2b69f8-b787-4147-ad2f-770902a152a4">
I see if you still use `-e` argument with realpath, it won't work

Reply:
> I see if you still use `-e` argument with realpath, it won't work

Going back to `realpath` and looping through possible `realpath` executables in the path to find GNU `realpath` in ff3c4562c8.